public/video-ui/styles/components/_detail.scss

```diff
@@ -9,6 +9,9 @@
     &__item {
       padding: 5px 10px;
       border-top: 1px solid $color600Grey;
+      width: 100%;
+      overflow: hidden;
```

Comment:
I've a feeling this is going to annoy people. We can leave it in for now though.